native/android/build.gradle

```diff
@@ -2,10 +2,10 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "29.0.3"
+        buildToolsVersion = "30.0.2"
         minSdkVersion = 21
-        compileSdkVersion = 29
-        targetSdkVersion = 29
+        compileSdkVersion = 30
+        targetSdkVersion = 30
```

Comment:
I suppose all of these changes come from the offical diff, right?

Reply:
Yes, see https://react-native-community.github.io/upgrade-helper/?from=0.64.2&to=0.65.1